pinot/regressors/biophysical_regressor.py

```diff
@@ -0,0 +1,87 @@
+# =============================================================================
+# IMPORTS
+# =============================================================================
+import torch
+import pinot
+import abc
+import math
+from pinot.regressors.base_regressor import BaseRegressor
+from pinot.regressors.gaussian_process_regressor import ExactGaussianProcessRegressor
+from pinot.regressors.neural_network_regressor import NeuralNetworkRegressor
```

Comment:
both of these imports are unused

Reply:
yeah I guess my design ideas about this class changed as I was doing it. Lint would catch all these issues, but has not run yet.